src/components/LoadingPlaceholder.vue

```diff
@@ -20,51 +20,50 @@
 -->
 
 <template>
-	<ul>
+	<div class="placeholder-main">
 		<!-- Placeholder animation -->
-		<svg class="placeholder-gradient">
+		<svg class="placeholder-gradient placeholder-gradient-regular">
 			<defs>
-				<linearGradient id="placeholder-gradient">
-					<stop offset="0%" :stop-color="light">
-						<animate attributeName="stop-color"
-							:values="`${light}; ${light}; ${dark}; ${dark}; ${light}`"
-							dur="2s"
-							repeatCount="indefinite" />
-					</stop>
-					<stop offset="100%" :stop-color="dark">
-						<animate attributeName="stop-color"
-							:values="`${dark}; ${light}; ${light}; ${dark}; ${dark}`"
-							dur="2s"
-							repeatCount="indefinite" />
-					</stop>
+				<linearGradient id="placeholder-gradient-regular">
+					<stop offset="0%" :stop-color="light" />
```

Comment:
You can't use variables there?

Reply:
for `0%` ?

Reply:
adjusted

Reply:
Lol no, for `stop-color="var(--color-blabla)"`

Reply:
Ah, ignore my comments, I think I misunderstood the code :thinking: